packages/react-icons/src/createIcon.tsx

```diff
@@ -66,13 +66,11 @@ export function createIcon({
 
       const hasTitle = Boolean(title);
       const heightWidth = getSize(size);
-      const baseAlign = -0.125 * Number.parseFloat(heightWidth);
-      const style = noVerticalAlign ? null : { verticalAlign: `${baseAlign}em` };
       const viewBox = [xOffset, yOffset, width, height].join(' ');
 
       return (
         <svg
-          style={style}
+          className={noVerticalAlign ? undefined : 'pf-svg-vertical-align'}
```

Comment:
To fix the few issues @mcoker mentioned destructure `className` prop and change to ```className={`${className}${noVerticalAlign ? '' : 'pf-svg-vertical-align'}`}```.